### PR TITLE
keep a changelog

### PR DESCRIPTION
**Pull request by techtangents**

### CODE REVIEW

**Review by tiny-james — APPROVED** [review]

**Review by jscasca — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,112 +1,240 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
 ## Unreleased
```
Comment (jscasca) [inline CHANGELOG.md:7]: ## 4.2.2 (2021-03-18)
### Added
* Event `ResizeEditor` to event handler
* Warning on multiple ediotrs with same Id

## 4.2.1 (2021-02-10)
### Added
* Beehive-flow release process
* Support for Angular 11

Comment (jscasca) [reply CHANGELOG.md:7]: Not sure why those lines were missing

Comment (techtangents) [reply CHANGELOG.md:7]: Where did you get these lines from? I can't see them in the git history.

Comment (jscasca) [reply CHANGELOG.md:7]: Sorry, my bad. They are not in history but it is a great time to add them :)

Comment (techtangents) [reply CHANGELOG.md:7]: Done.

**Pull request merged**